### DISCUSSION TITLE
This node requires Node-RED custom integration version 3.0.0+  to be installed in Home Assistant for it to function

### DISCUSSION THREAD

**Discussion by wakefib** [zachowj/node-red-contrib-home-assistant-websocket#1392] [Q&A]
i keep getting this message, everything is up to date!. It seems to happen after i have updated home assistant. I have re downloaded and installed nodered custom. 
Really frustrating as the device node has no devices..

**Comment by zachowj** [comment]
Next time it happens check the [diagnostics button](https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/diagnostics.html) to see what version node-RED thinks is installed.

**Comment by wakefib** [comment]
I solved this, will a good work around.

running node red via homeassistant.local:1880

Everything is working, also prior to this inject buttons stopped working

the flow before still worked it was just i could not select any devices in ha, while loading node red from within ha

**Comment by wakefib** [comment]
Fixed!
